src/App.js

```diff
@@ -1,25 +1,30 @@
-import logo from './logo.svg';
 import './App.css';
+import SignButton from './components/Buttons/BasicSigns';
+import NumberButton from './components/Buttons/NumberButton';
+import CalcInput from './components/CalcInput';
+import { numbers, signs } from './static/button';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          <code>src/App.js</code>
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="w-2/4 items-center">
+      <div className="w-full">
+        <CalcInput />
+      </div>
+      <div className="flex">
+        <div className="grid  grid-cols-3  w-4/5 self-start">
+          {signs.secondary.map((sign) => (
+            <SignButton key={sign.id} sign={sign.name} type="sec" />
+          ))}
+          {numbers.map((number) => (
+            <NumberButton key={number.id} number={number.name} />
+          ))}
+        </div>
+        <div className="flex flex-col w-1/5">
+          {signs.basic.map((sign) => (
+            <SignButton key={sign.id} sign={sign.name} type="basic" />
+          ))}
+        </div>
+      </div>
     </div>
   );
 }
```

Comment:
- [ ] <b>REQUIRED:</b> All of those components your imported here are supposed to be imported and used in `Calculator` component instead. and then use `Calculator` component in here. Kindly do that to comply with project requirements.

Reply:
Okay, I will implement that but kindly does the name of components matter ??